cogs/vxtwitconverter/info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name" : "VxTwitConverter",
+    "author" : ["JadeRabbit#9310"],
+    "short" : "twitter embed fix when?",
+    "description" : "Converts Twitter video embeds into vxTwitter links instead",
```

Comment:
I think many people know what VxTwitter is, but it's good to add a short text in hear saying "VxTwitter provides better Twitter video embeds."